### PR TITLE
Add Residential dock to Dallus

### PR DESCRIPTION
**Pull request by Brimcon**
Adds Residential Shuttle to ARFS Dallus
Adds Residential DMM's to maps/arfs
Fixes some Maint areas.
Adds best (annoying) bird.

### CODE REVIEW

File: maps/Arfs/arfs.dm [hunk 1]
```diff
@@ -11,7 +11,9 @@
 	#include "arfs-6-asteroid.dmm"
 	#include "arfs-7-mining.dmm"
 	#include "arfs-8-underdark.dmm"
-
+	#include "residential/_residential_arf.dm"
```
Comment (TalkingCactus) [inline maps/Arfs/arfs.dm]: Separate the map files from the code files please

File: maps/Arfs/arfs.dm [hunk 2]
```diff
@@ -11,6 +11,7 @@
 	#include "arfs-6-asteroid.dmm"
 	#include "arfs-7-mining.dmm"
 	#include "arfs-8-underdark.dmm"
+
 	#include "residential/_residential_arf.dm"
 	#include "residential/residential_arf.dmm"
```
Comment (TalkingCactus) [inline maps/Arfs/arfs.dm]: it's still here O_O

Comment (Brimcon) [reply maps/Arfs/arfs.dm]: IT SHOULDN'T BE

Comment (TalkingCactus) [reply maps/Arfs/arfs.dm]: move it below the underdark map file

File: maps/Arfs/residential/_residential.dm [hunk 3]
```diff
@@ -0,0 +1,210 @@
+// -- Shuttle -- //
```
Comment (TalkingCactus) [inline maps/Arfs/residential/_residential.dm]: This entire file can be deleted, as it's not in use anywhere and is just a copy of _residential_arfs.dm

**Review by TalkingCactus — APPROVED** [review]

**Pull request merged**